testing_utils/sorter_stress_test/sorter_stress.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
```

Comment:
should we start unified sorter in all current integration test cases

Reply:
Unified Sorter has been set to be the default sorter in this PR.